common/app/views/support/HtmlCleaner.scala

```diff
@@ -698,9 +698,16 @@ case class AtomsCleaner(atoms: Option[Atoms], shouldFence: Boolean = true, amp:
         if(atomData.isInstanceOf[MediaAtom]){
           atomContainer.addClass("element-atom--media")
         }
-        val html = views.html.fragments.atoms.atom(atomData, shouldFence, amp, mediaWrapper).toString()
-        bodyElement.remove()
-        atomContainer.append(html)
+
+        if (ReaderAnswersDeliveryMechanism.isSwitchedOn) {
+          val html = views.html.fragments.atoms.atom(atomData, shouldFence, amp, mediaWrapper, true).toString()
+          bodyElement.remove()
+          atomContainer.append(html)
+        } else {
+          val html = views.html.fragments.atoms.atom(atomData, shouldFence, amp, mediaWrapper, false).toString()
+          bodyElement.remove()
+          atomContainer.append(html)
+        }
```

Comment:
can we pass the switch value as the last param rather than using a `if/else`:
```
val html = views.html.fragments.atoms.atom(atomData, shouldFence, amp, mediaWrapper, ReaderAnswersDeliveryMechanism.isSwitchedOn).toString()
bodyElement.remove()
atomContainer.append(html)
```

Reply:
Can the switch test be moved inside the storyquestions template entirely instead of threading it through calls like that?

Reply:
Hmmm not sure I'll take a look.

Reply:
So do you mean moving it here like this @regiskuckaertz ?
```
if (OurFeatureTest.isSwitchedOn) {
      render specific html for test
}
```

Reply:
Oui 😄